java/java.disco/src/org/netbeans/modules/java/disco/AdvancedPanel.java

```diff
@@ -378,12 +395,12 @@ private void eaCheckBoxActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FI
     }//GEN-LAST:event_eaCheckBoxActionPerformed
 
     private void filterChanged() {
-        updateData( (Distribution) distributionComboBox.getSelectedItem(),
-                    (Integer) versionComboBox.getSelectedItem(),
-                    (Architecture) architectureComboBox.getSelectedItem(),
-                    latestCheckBox.isSelected() ? Latest.OVERALL : Latest.ALL_OF_VERSION,
-                    (PackageType) packageTypeComboBox.getSelectedItem(),
-                    eaCheckBox.isSelected());
+        updateData((Distribution) distributionComboBox.getSelectedItem(),
+                (Integer) versionComboBox.getSelectedItem(),
+                (Architecture) architectureComboBox.getSelectedItem(),
+                latestCheckBox.isSelected() ? Latest.OVERALL : Latest.ALL_OF_VERSION,
+                (PackageType) packageTypeComboBox.getSelectedItem(),
+                eaCheckBox.isSelected());
```

Comment:
since there are only whitespace changes here, I was wondering if you wanted to call the utility getters you made from here but you forgot?

Reply:
No, just being guilty of my own pet hate. Must have been a little over zealous on what regions I selected to auto-format.